main.py

```diff
@@ -141,14 +143,44 @@ def login(driver, username, password):
     email_input = driver.find_element_by_xpath("//input[@name='emailAddress']")
     email_input.clear()
     email_input.send_keys(username)
+    
     password_input = driver.find_element_by_xpath("//input[@name='password']")
     password_input.clear()
     password_input.send_keys(password)
 
     LOGGER.info("Logging in")
     driver.find_element_by_xpath("//input[@value='SIGN IN']").click()
-    wait_until_visible(driver=driver, xpath="//a[@data-path='myAccount:greeting']")
+    
+    sleep_val = 3
+    num_retries_attempted = 0
+    num_retries = 5
+    while True:
+        try:
+            time.sleep(sleep_val)
+            xpath = "/html/body/div[2]/div[3]/div[3]/div/div[2]/input"
+            wait_until_visible(driver=driver, xpath=xpath, duration=5)
+            driver.find_element_by_xpath(xpath).click()
+        
+            password_input = driver.find_element_by_xpath("//input[@name='password']")
```

Comment:
So the email input always stays as-is (with the previously entered data) after the error dialog goes away?

Reply:
Yes, only password disappears (at least on Firefox)